.github/workflows/ci.yml

```diff
@@ -54,9 +55,11 @@ jobs:
         node-version: ${{ env.NODE_VERSION }}
         cache: pnpm
     - name: Install Dependencies
-      run: rm pnpm-lock.yaml && pnpm install
+      run: rm ../pnpm-lock.yaml && pnpm install
```

Comment:
Apparently, pnpm has an option `--no-lockfile` which you can use instead of removing the lockfile. I saw it in other repos.

Reply:
Done